Samples/Forms/LibVLCSharp.Forms.MediaElement/LibVLCSharp.Forms.MediaElement.Sample/Sample.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0"?>
```

Comment:
Looks like a generated file in the wrong place

Reply:
You are right, it must be deleted because it is generated into the bin folder.